syntax="proto3";
package filerebuild;

service WiriteFile {
    rpc SendFile (FileInfoRequest) returns (FileResponse) {}
}

message FileInfoRequest {
    string file_path = 1;
    string file_name = 2;
    bytes content = 3;
}

message FileResponse {
    string ret_code = 1;
    string info = 2;
    string content = 3;
}